README.md

```diff
@@ -34,7 +34,8 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | vpc_id | VPC ID. | string | - | yes |
 | wd_ecs_task_count | Number of ECS tasks to create. | string | `1` | no |
 | wd_instance_type | Waggle Dance instance type, possible values: `ecs`,`k8s`. | string | `ecs` | no |
-| waggledance_version | Waggle Dance version to install on EC2 nodes | string | `3.3.2` | no |
+| waggledance_version | Waggle Dance version to install on EC2 nodes. | string | `3.3.2` | no |
+| waggledance_logs_retention_days | Log retention in days for the Waggle Dance cloudwatch log group. | string | `30` | no |
```

Comment:
```suggestion
| waggledance_logs_retention_days | Log retention in days for the Waggle Dance CloudWatch log group. | string | `30` | no |
```

Reply:
actually @rpoluri suggested adding cloudwatch in the variable name so to distinguish it from the more generic "log" type variables which are associated with logs on S3. This is for apiary-data-lake though. Here we don't store the logs on S3.